components/style/properties/helpers.mako.rs

```diff
@@ -240,12 +245,16 @@
                     |value| {
                         if let Some(ref mut cascade_info) = *cascade_info {
                             cascade_info.on_cascade_property(&declaration,
-                                                             &value);
+                                                             value);
                         }
                         <% maybe_wm = ", wm" if property.logical else "" %>
                         match *value {
                             DeclaredValue::Value(ref specified_value) => {
-                                let computed = specified_value.to_computed_value(context);
+                                % if property.boxed:
+                                    let computed = (*specified_value).to_computed_value(context);
```

Comment:
I can’t reproduce that error… (after reverting the WIP commit which does error) Could you copy/paste it?